### PR TITLE
add some popular http library & scraping framework

### PR DESCRIPTION
**Pull request by Zerorigin**
1. Add some popular http client library into scripting-user-agents.data
```
# https://docs.aiohttp.org/en/stable/
# User-Agent from: https://github.com/aio-libs/aiohttp/blob/3.0/aiohttp/http.py
# User-Agent: Python/VERSION aiohttp/VERSION
aiohttp/


# https://pkg.go.dev/net/http
# old User-Agent from: https://codereview.appspot.com/7532043
 # new User-Agent from: https://cs.opensource.google/go/go/+/refs/tags/go1.10rc1:src/net/http/request.go;l=462
# User-Agent: Go http package
# User-Agent: Go 1.1 package http
# User-Agent: Go-http-client/VERSION
Go http package
Go 1.1 package http
Go-http-client/


# https://www.python-httpx.org/
# User-Agent: python-httpx/VERSION
python-httpx/


# https://kong.github.io/unirest-java/
# User-Agent: unirest-java/VERSION
unirest-java/
```

2. Add some popular scraping framework into crawlers-user-agents.data
```
# https://ache.readthedocs.io/en/latest/
# User-Agent from: https://github.com/VIDA-NYU/ache/blob/master/ache-tools/src/main/java/achecrawler/tools/GenerateTLDLists.java
# User-Agent: (Mozilla/5.0 (compatible; ACME/VERSION; +OPERATOR_CONTACT_URL; +OPERATOR_CONTACT_EMAIL)
ACHE/


# http://go-colly.org/
# User-Agent from: https://github.com/gocolly/colly/blob/master/colly.go
# User-Agent: colly - https://github.com/gocolly/colly/v2
colly -


# https://github.com/yasserg/crawler4j
# User-Agent from: https://github.com/yasserg/crawler4j#user-agent-string
# User-Agent: crawler4j (https://github.com/yasserg/crawler4j/)
crawler4j


# https://github.com/internetarchive/heritrix3
# User-Agent from: https://github.com/internetarchive/heritrix3/blob/master/modules/src/main/java/org/archive/modules/CrawlMetadata.java
# User-Agent: "Mozilla/5.0 (compatible; heritrix/VERSION +OPERATOR_CONTACT_URL)
heritrix/


# http://docs.seattlerb.org/mechanize/Mechanize.html
Mechanize


# https://nutch.apache.org/
# User-Agent from: https://github.com/apache/nutch/blob/2.1/src/plugin/lib-http/src/java/org/apache/nutch/protocol/http/api/HttpBase.java
# User-Agent: NutchCVS/VERSION (Nutch; http://lucene.apache.org/nutch/bot.html; nutch-agent@lucene.apache.org)
NutchCVS/


# https://docs.pyspider.org/en/latest/
# User-Agent from: https://docs.pyspider.org/en/latest/tutorial/AJAX-and-more-HTTP/#user-agent
# User-Agent: pyspider/VERSION (+http://pyspider.org/)
pyspider/


# https://scrapy.org/
# User-Agent from: https://docs.scrapy.org/en/latest/topics/settings.html?highlight=user-agent#user-agent
# User-Agent: Scrapy/VERSION (+https://scrapy.org)
Scrapy/
```

### CODE REVIEW

File: rules/scripting-user-agents.data [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
 # http://search.cpan.org/~opera/HTTP-DAV/DAV.pm
 dav.pm/v
 # http library
+# https://pkg.go.dev/net/http
+# User-Agent: Go http package
+# User-Agent: Go 1.1 package http
+# User-Agent: Go-http-client/VERSION
+Go http package
+Go 1.1 package http
+Go-http-client/
```
Comment (fzipi) [inline rules/scripting-user-agents.data:17]: Hmmm... 🤔 what happenss with Prometheus and similar software that is doing monitoring?

Comment (Zerorigin) [reply rules/scripting-user-agents.data:17]: > Hmmm... 🤔 what happenss with Prometheus and similar software that is doing monitoring?

I think the WAF should be used to protect the external interface, not the internal interface.

Comment (dune73) [reply rules/scripting-user-agents.data:17]: Why should `Go` get better treatmen than `libwww-perl` and python?

Comment (fzipi) [reply rules/scripting-user-agents.data:17]: Just because this is PL1 and we try to avoid FPs as much as possible.

Comment (lifeforms) [reply rules/scripting-user-agents.data:17]: I don’t think it should get a special treatment. It’s very easy to change the User-Agent header after 2 hours into Go (example: https://github.com/lifeforms/httpcheck/blob/master/httpcheck/init.go#L35) so any serious client should change it...

**Comment by dune73** [comment]
Have not heard any additional comment on this PR. Looks good to me, merging now.

Thank @Zerorigin for your welcome contribution. Well done.

**Pull request merged**